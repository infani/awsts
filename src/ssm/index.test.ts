import { SSM } from './index';

describe('index', () => {
  it('index', async () => {
    const res = await SSM.GetParameter('TestGetParameter');
    expect(res?.Name).toMatchSnapshot();
  });
});
