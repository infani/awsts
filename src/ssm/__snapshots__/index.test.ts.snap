// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index index 1`] = `"TestGetParameter"`;
